docker-app/qfieldcloud/core/views/files_views.py

```diff
@@ -125,6 +141,27 @@ def has_permission(self, request, view):
         return False
 
 
+@extend_schema_view(
+    get=extend_schema(
+        description="Download a file from a project",
+        responses={
+            (200, "text/html"): OpenApiTypes.BINARY,
+        },
+    ),
+    post=extend_schema(
+        description="Upload a file to the project",
+        parameters=[
+            OpenApiParameter(
+                "file",
+                OpenApiTypes.STR,
+                OpenApiParameter.QUERY,
+                required=True,
+                description="Path to the file to upload",
+            )
```

Comment:
```suggestion
            OpenApiParameter(
                "file",
                OpenApiTypes.BINARY,
                OpenApiParameter.QUERY,
                required=True,
                description="File to be uploaded",
            )
```

Reply:
This is file, did you test actually making requests using the Swagger UI?

Reply:
It's on the menu -- once you confirm the API docs suits you :)